# Archaic cas names

Many CRISPR-associated genes have been renamed, sometimes more than once. Sequences databases often retain an archaic name, making searching for particular genes tricky. This repo exists to keep track of the various alternatives for each *cas* gene. In cases where there is no consensus in the community, this database will prefer the more consistent naming scheme. 

### Contributing

Please open an issue or submit a pull request (with references) if you feel there should be any additions or modifications to this list. You can also just email [jim@rybarski.com](mailto:jim@rybarski.com).

### Names

Each name is given with the reference where it was first used.

| Current name | Previous name(s) |
| --- | --- |
| cas1 [1] | cas1a [2], cas1b [2] |
| cas2 [1] | cas6 [2] |
| cas3 [1] | - |
| cas4 [1] | csa1 [3] |
| cas5 [3] | casD [4], cas5a [3], cas5d [3], cas5e [3], cas5h [3], cas5p [5\*], cas5t [3], cmx5 [5\*], csc1 [5\*], csy2 [3], csf3 [5\*], csm4 [3], csx10 [5\*], cmr3 [3] |
| cas6 [3] | cmx6 [5\*] |
| cas6e [5] | casE [4], cse3 [3] |
| cas6f [5] | csy4 [3] |
| cas7 [5] | casC [4], csa2 [3], csd2 [3], cse4 [3], csh2 [3], csp1 [5\*], cst2 [3], csm3 [3], cmr4 [3], csc2 [5\*], csy3 [3], csf2 [5\*], csm5 [3], cmr1 [3], cmr6 [3] |
| cas8 [5] | csf1 [5\*] |
| cas8a1 [5] | cmx1 [5\*], cst1 [3], csx8 [5\*], csx13 [5\*], CXXC-CXXC [5\*] |
| cas8a2 [5] | csa4 [3], csx9 [5\*] |
| cas8b [5] | csh1 [3], TM1802 [5\*] |
| cas8c [5] | csd1 [3], csp2 [5\*] |
| cas8e [7] | casA [4], cse1 [3] |
| cas8f [7] | csy1 [3] |
| cas9 [5] | csn1 [3], csx12 [5\*], cas5 [2] |
| cas10 [5] | cmr2 [3], csm1 [3], csx1 [3] |
| cas10d [5] | csc3 [5\*] |
| cas11 [6] | casB [4], cse2 [3], csm2 [3], cmr5 [3], csa5 [3] |
| cas12a [9] | cpf1 [8] |
| cas12b [9] | c2c1 [10] |
| cas12c [9] | c2c3 [10] |
| cas12d [9] | casY [11] |
| cas12e [9] | casX [11] |
| cas12f [13] | cas14 [12] |
| cas12g [14] | - |
| cas12h [14] | - |
| cas12i [14] | - |
| cas12j [15] | casΦ/casphi [15] |
| cas12k [17] | c2c5 [16] |
| cas12l\*\* [18, 19, 20] | casπ/caspi [20] |
| cas12m [21] | - |
| cas13a [16] | C2c2 [10] |
| cas13b [16] | C2c6 [16\*]  |
| cas13c [16] | C2c7 [16\*] |
| cas13d [22, 23] | - |

\* We are unable to find any published work that uses these designations, but they are cited as the archaic names of these genes.  
\*\* Three groups have independently conferred the designation "cas12l" to a set of genes, with no apparent connection.

### References

| \# | Reference | DOI |
| --- | --- | --- |
| 1 | Jansen et al. *Identification of genes that are associated with DNA repeats in prokaryotes.* Mol. Microbio. (2002) | [10.1046/j.1365-2958.2002.02839.x](https://doi.org/10.1046/j.1365-2958.2002.02839.x) |
| 2 | Bolotin et al. *Clustered regularly interspaced short palindrome repeats (CRISPRs) have spacers of extrachromosomal origin.* Microbiology (2005) | [10.1099/mic.0.28048-0](https://doi.org/10.1099/mic.0.28048-0) |
| 3 | Haft et al. *A Guild of 45 CRISPR-Associated (Cas) Protein Families and Multiple CRISPR/Cas Subtypes Exist in Prokaryotic Genomes.* PLOS Comp. Bio. (2005) | [10.1371/journal.pcbi.0010060](https://doi.org/10.1371/journal.pcbi.0010060) |
| 4 | Brouns and Jore et al. *Small CRISPR RNAs Guide Antiviral Defense in Prokaryotes.* Science (2008) | [10.1126/science.1159689](https://doi.org/10.1126/science.1159689) |
| 5 | Makarova et al. *Evolution and classification of the CRISPR–Cas systems.* Nat. Rev. Microbio. (2011) | [10.1038/nrmicro2577](https://doi.org/10.1038/nrmicro2577) |
| 6 | Makarova et al. *Unification of Cas protein families and a simple scenario for the origin and evolution of CRISPR- Cas systems.* Bio. Direct (2011) | [10.1186/1745-6150-6-38](https://doi.org/10.1186/1745-6150-6-38) |
| 7 | Makarova et al. *CRISPR-Cas Systems and Cas Protein Families.* Protein Families: Relating Protein Sequence, Structure, and Function (2013) | [10.1002/9781118743089.ch13](https://doi.org/10.1002/9781118743089.ch13) |
| 8 | Zetsche and Gootenberg et al. *Cpf1 Is a Single RNA-Guided Endonuclease of a Class 2 CRISPR-Cas System.* Cell (2015) | [10.1016/j.cell.2015.09.038](https://doi.org/10.1016/j.cell.2015.09.038) |
| 9 | Koonin et al. *Diversity, classification and evolution of CRISPR-Cas systems.* Curr. Opin. Microbiol. (2017) | [10.1016/j.mib.2017.05.008](https://doi.org/10.1016/j.mib.2017.05.008) |
| 10 | Shmakov and Abudayyeh et al. *Discovery and Functional Characterization of Diverse Class 2 CRISPR-Cas Systems.* Mol. Cell (2015) | [10.1016/j.molcel.2015.10.008](https://doi.org/10.1016/j.molcel.2015.10.008) |
| 11 | Burstein and Harrington et al. *New CRISPR–Cas systems from uncultivated microbes.* Nature (2016) | [10.1038/nature21059](https://doi.org/10.1038/nature21059) |
| 12 | Harrington and Burstein et al. *Programmed DNA destruction byminiature CRISPR-Cas14 enzymes.* Science (2018) | [10.1126/science.aav4294](https://doi.org/10.1126/science.aav4294) |
| 13 | Makarova et al. *Evolutionary classification of CRISPR–Cas systems: a burst of class 2 and derived variants.* Nat. Rev. Microbio. (2020) | [10.1038/s41579-019-0299-x](https://doi.org/10.1038/s41579-019-0299-x) |
| 14 | Yan and Hunnewell et al. *Functionally diverse type V CRISPR-Cas systems.* Science (2018) | [10.1126/science.aav7271](https://doi.org/10.1126/science.aav7271) |
| 15 | Al-Shayeb and Sachdeva et al. *Clades of huge phages from across Earth's ecosystems.* Nature (2020) | [10.1038/s41586-020-2007-4](https://doi.org/10.1038/s41586-020-2007-4) |
| 16 | Shmakov et al. *Diversity and evolution of class 2 CRISPR–Cas systems.* Nat. Rev. Microbio. (2017) | [10.1038/nrmicro.2016.184](https://doi.org/10.1038/nrmicro.2016.184) |
| 17 | Strecker et al. *RNA-guided DNA insertion with CRISPR-associated transposases.* Science (2019) | [10.1126/science.aax9181](https://doi.org/10.1126/science.aax9181) |
| 18 | Goltsman et al. *Novel Type V-A CRISPR Effectors Are Active Nucleases with Expanded Targeting Capabilities.* CRISPR Journal (2020) | [10.1089/crispr.2020.0043](https://doi.org/10.1089/crispr.2020.0043) |
| 19 | Urbaitis, Gasiunas, and Young et al. *A new family of CRISPR-type V nucleases with C-rich PAM recognition.* EMBO Reports (2022) | [10.15252/embr.202255481](https://doi.org/10.15252/embr.202255481) |
| 20 | Sun, Li, Chen and Zhang et al. *The compact Casπ (Cas12l) ‘bracelet’ provides a unique structural platform for DNA manipulation.* Cell Research (2023) | [10.1038/s41422-022-00771-2](https://doi.org/10.1038/s41422-022-00771-2) |
| 21 | Wu and Mohanraju et al. *The miniature CRISPR-Cas12m effector binds DNA to block transcription.* Mol. Cell (2022) | [10.1016/j.molcel.2022.11.003](https://doi.org/10.1016/j.molcel.2022.11.003) |
| 22 | Yan, Cheng and Scott et al. *Cas13d Is a Compact RNA-Targeting Type VI CRISPR Effector Positively Modulated by a WYL-Domain-Containing Accessory Protein.* Mol. Cell (2018) | [10.1016/j.molcel.2018.02.028](https://doi.org/10.1016/j.molcel.2018.02.028) |
| 23 | Konermann et al. *Transcriptome Engineering with RNA-Targeting Type VI-D CRISPR Effectors.* Cell (2018) | [10.1016/j.cell.2018.02.033](https://doi.org/10.1016/j.cell.2018.02.033) |
